demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,172 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader,
+  useMeetingManager,
+} from 'amazon-chime-sdk-component-library-react';
+
+import { MediaStatsList } from '../../components/MediaStatsList';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const {
+    audioPacketsSentFractionLossPercent,
+    audioPacketsReceivedFractionLossPercent,
+    availableIncomingBandwidth,
+    availableOutgoingBandwidth,
+    videoStreamMetrics
+  } = useMediaStreamMetrics();
+
+  const isLocalAudioActive =
+  audioVideo &&
+  audioPacketsSentFractionLossPercent !== null &&
+  audioPacketsReceivedFractionLossPercent !== null;
+
+  const meetingManager = useMeetingManager();
+  const localAttendeeId = meetingManager.meetingSession?.configuration.credentials?.attendeeId;
+  const localVideoStreamMetrics = localAttendeeId ? videoStreamMetrics[localAttendeeId] : {};
+  const ssrcArray = localVideoStreamMetrics ? Object.keys(localVideoStreamMetrics) : [];
+
+  const isLocalVideoActive = audioVideo && ssrcArray.length !== 0;
+  const hasBandwidthInfo =
+    availableIncomingBandwidth !== null && availableOutgoingBandwidth !== null;
+  return (
+    <StyledMediaMetricsWrapper>
+      {isLocalAudioActive && (
+        <>
+          <PopOverHeader title={'Audio statistics'} />
+          <MediaStatsList>
+            <MetricItem metricName="" metricValues={['Up', 'Down']} />
+            <MetricItem
+              metricName="1s Loss"
+              metricValues={[
+                audioPacketsSentFractionLossPercent !== null ? audioPacketsSentFractionLossPercent.toString() : "",
+                audioPacketsReceivedFractionLossPercent !== null ? audioPacketsReceivedFractionLossPercent.toString(): ""
+              ]}
+            />
+          </MediaStatsList>
+        </>
+      )}
+      {isLocalVideoActive && (
+        <>
+          <PopOverHeader title={'Video statistics'} />
+          <MediaStatsList>
+            <MetricItem
+              metricName="Bit rate (kbps)"
+              metricValues={
+                ssrcArray.map(ssrc => {
+                  return localVideoStreamMetrics[ssrc].videoUpstreamBitrate
+                    ? Math.trunc(
+                        localVideoStreamMetrics[ssrc].videoUpstreamBitrate /
+                          1000
+                      ).toString()
+                    : '';
+                })
+              }
+            />
+            <MetricItem
+              metricName="Packets Sent"
+              metricValues={
+                ssrcArray &&
+                ssrcArray.map(ssrc => {
```

Comment:
Same comment what I said for `srcArray.map` here :)

Reply:
Good catch. will remove this.